src/Telemetry/Exit_Interview_Subscriber.php

```diff
@@ -48,6 +82,17 @@ public function ajax_exit_interview() {
 		wp_send_json_success();
 	}
 
+	/**
+	 * Updates the "deactivate" link on the plugin so that it doesn't actually trigger the deactivation.
```

Comment:
```suggestion
	 * Adds an <i> element after the "deactivate" link on the plugin list table so that it can be targeted by JS to trigger the interview modal.
```